covfee/client/journey/journey.tsx

```diff
@@ -1,14 +1,13 @@
-import * as React from "react"
-import styled from "styled-components"
-import { generatePath } from "react-router"
 import { ArrowRightOutlined, PlusOutlined } from "@ant-design/icons"
-import { Row, Col, Typography, Menu, Button, Modal, Progress } from "antd"
+import { Button, Menu, Modal, Progress, Row } from "antd"
```

Comment:
There should be no changes in this file, I think it'd be better to undo them and then merge #38 into this branch

Reply:
You are right. I'll undo them, and hopefully #38 and this PR will get merged at the same time.

Reply:
Changes in journey.tsx were removed.